index.sh

```diff
@@ -243,18 +243,29 @@ if [ "${altcmd:-x}" != "x" ] ; then
 |     update-alternatives detected     |
 \\=====================================/
 
+(Notice: getmicro must be run as root, e.x.: curl https://getmic.ro | sudo sh
```

Comment:
As discussed below, I think we can just improve how we handle non-root users here; hopefully that makes this extra warning unneccesary.

Reply:
I agree. This message is no longer necessary.